app/modules/intelligence/agents/chat_agents/pydantic_agent.py

```
@@ -107,12 +144,34 @@ def _create_task_description(
         if isinstance(ctx.node_ids, str):
             ctx.node_ids = [ctx.node_ids]
 
+        # Add image context information
+        image_context = ""
+        if ctx.has_images():
+            all_images = ctx.get_all_images()
+            image_details = []
+            for attachment_id, image_data in all_images.items():
```

Ruff (0.11.9)

Loop control variable `attachment_id` not used within loop body

Rename unused `attachment_id` to `_attachment_id`

(B007)